EricGray/homework.js

```diff
@@ -0,0 +1,386 @@
+//Part 1: A Cards Toolkit!>
+
+
+
+var suitARR = [" Hearts", " Diamonds", " Spades", " Clubs"];
+var rankARR = ["Ace ", "Two ", "Three ", "Four ", "Five ", "Six ", "Seven ", 
+"Eight ", "Nine ", "Ten ", "Jack ", "Queen ", "King "];
+
+function checkValidID(id) {
+	if (id < 0 || id > 51 || id%1 !== 0) {
+		alert("Sorry, your number is out of range. Only whole numbers 0-51, please!)");
+		return NaN;
+			} else if (typeof id === "string" || typeof id === "boolean"
+			 || typeof id === "undefined") {
+		alert("Sorry, only whole numbers 0-51, please!");
+		return NaN;
+	} else {
+		return id;
+	} 
+;}
+
```

Comment:
Your code works technically, but anytime you are repeating yourself more than once, you should pull the code into a function, and call the function in place of the code.